docs/source/markdown/options/network.md

```diff
@@ -9,32 +9,38 @@ Set the network mode for the <<container|pod>>.
 Valid _mode_ values are:
 
 - **bridge[:OPTIONS,...]**: Create a network stack on the default bridge. This is the default for rootful containers. It is possible to specify these additional options:
-  - **alias=name**: Add network-scoped alias for the container.
-  - **ip=IPv4**: Specify a static ipv4 address for this container.
-  - **ip=IPv6**: Specify a static ipv6 address for this container.
-  - **mac=MAC**: Specify a static mac address for this container.
-  - **interface_name**: Specify a name for the created network interface inside the container.
+  - **alias=**_name_: Add network-scoped alias for the container.
+  - **ip=**_IPv4_: Specify a static ipv4 address for this container.
+  - **ip=**_IPv6_: Specify a static ipv6 address for this container.
+  - **mac=**_MAC_: Specify a static mac address for this container.
+  - _interface\_name_: Specify a name for the created network interface inside the container.
```

Comment:
pre-existing but this is incorrect, it should be `**interface_name=**_name_:`

Reply:
Fixed in the next push.